### ISSUE TITLE
export table to json

### ISSUE THREAD

**Issue by bvbg1** [aws-samples/amazon-textract-textractor#174]
I can see support for exporting to csv, Pandas DataFrame and xls here:
https://aws-samples.github.io/amazon-textract-textractor/notebooks/table_data_to_various_formats.html

Is there a way to export the table data to json? 
Or do I have to export as DataFrame first and then convert to json by using pandas?

**Comment by ThomasDelteil** [comment]
It depends what you try to achieve, in a way, the Textract API response is already a json that contains the tables information. So if you filter for only the TABLE and CELL and MERGED_CELL blocks, you have the table data there. However it is heavy on the geometric information of the data. 

If you mostly care about the data contained in the table rather than its geometric information, I would go the route you suggest with pandas and export to json. A more lightweight option would be to export to csv format and then use a csv_reader to parse the data into lists.

You can also easily just go through the table and add it to a matrix:

```
output = {}
for table in document.tables:
    table_data = [[[''] for _ in range(table.column_count)] for _ in range(table.row_count)]
    for cell in table.table_cells:
        table_data[cell.row_index-1][cell.col_index-1] = cell.text
    output[table.id] = table_data
```

then you can export output to json using `json.dumps`

**Comment by tb102122** [comment]
or can just submit a new PR with a new function
https://github.com/aws-samples/amazon-textract-textractor/blob/af44b14942f133d845d67cf368a413e355785548/textractor/entities/table.py#L405
the only line you need to change is https://github.com/aws-samples/amazon-textract-textractor/blob/af44b14942f133d845d67cf368a413e355785548/textractor/entities/table.py#L411 with to_json()
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_json.html

**Comment by bvbg1** [comment]
Thanks!
@ThomasDelteil does it put the blocks in order by y-axis automatically?
Also I want to run this in Lambda, does the layer (https://aws-samples.github.io/amazon-textract-textractor/using_in_lambda.html) come with pandas included?

**Comment by ThomasDelteil** [comment]
@bvbg1 yes because the data will be located where the row column index is, which is geometric by nature.

**Comment by bvbg1** [comment]
> Also I want to run this in Lambda, does the layer (https://aws-samples.github.io/amazon-textract-textractor/using_in_lambda.html) come with pandas included?

To answer my own question, it doesn't come with pandas included.

**Comment by bvbg1** [comment]
@Belval Is it possible to get a precompiled Lambda layer with pandas included?
